ci/dockerfiles/Dockerfile.pip-minimal

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10.8
```

Comment:
Could we use slim?

Reply:
Oh nice idea. Would have to do a bit more research though - the slim packages don't include compilers. Could get them with `apt install build-essential` but that adds 312 MB to the image size, so defeats the purpose a bit

Reply:
OK this was an awesome idea. So if you just install gcc and g++ you get from the current 1.06 GB image size down to 475 MB